server/util/hash/hash.go

```diff
@@ -7,10 +7,24 @@ import (
 	"unsafe"
 )
 
+// String returns the hex representation of the SHA256 hash of the given string.
 func String(input string) string {
 	return fmt.Sprintf("%x", sha256.Sum256([]byte(input)))
 }
 
+// Strings returns a hash of the concatenated string hashes.
+//
+// Note: This is not the same as the hash of the concatenated strings. The
+// returned hash is not necessarily useful as a content hash, but may be useful
+// as a deduping key.
+func Strings(input ...string) string {
```

Comment:
This will generate hashes with length proportional to the size of the input array, which users of this library might find confusing (and, for some use cases, not performant). Is there a reason to not just concatenate the strings and return the SHA256 of that for now?